### DISCUSSION TITLE
I've been trying to set up a simple signature auth flow through V1 and I can't get it working.

### DISCUSSION THREAD

**Discussion by GrahamMcBain** [WalletConnect/walletconnect-monorepo#1617] [Developer Support]
Does this have to do with the v2 switch? Here let me send you two looms to show you what I mean
https://www.loom.com/share/ed3064027cbd40db807888381ea19538 

https://www.loom.com/share/75c744950a7b459aaa2a2ae729fd258e

I'm building a smart contract wallet using V1, Do you have any advice?

**Comment by Brentably** [comment]
Thanks for asking this question for me, Graham :)

**Comment by Brentably** [comment]
Hey! I have another hint that might help diagnose the problem. When I console.log the connector:

console.log("init new connector")
 const connector = await new WalletConnect({
  uri: uri,
  // bridge: "https://bridge.walletconnect.org"
  // ,
  clientMeta: {
    description: "WalletConnect Developer App",
    url: "https://walletconnect.org",
    icons: ["https://walletconnect.org/walletconnect-logo.png"],
    name: "WalletConnect",
  }
})
console.log(connector)

It returns this connection with a different client meta. And this account address I'm unfamiliar with... it's referenced in the documentation but not in my code. That's really strange but maybe this helps diagnose the issue.

![hintGithub](https://user-images.githubusercontent.com/46073594/201391675-d1ee8ea0-40c7-4d36-92bd-ac42a3860f09.jpg)
